apps/okreads-e2e/src/specs/search-books.spec.ts

```diff
@@ -16,12 +16,19 @@ describe('When: Use the search feature', () => {
     expect(items.length).toBeGreaterThan(1);
   });
 
-  xit('Then: I should see search results as I am typing', async () => {
+  it('Then: I should see search results as I am typing', async () => {
     await browser.get('/');
     await browser.wait(
       ExpectedConditions.textToBePresentInElement($('tmo-root'), 'okreads')
     );
 
     // TODO: Implement this test!
+    const form = await $('form');
+    const input = await $('input[type="search"]');
+    await input.sendKeys('javas');
+    await(setTimeout(() => {}, 600));
```

Comment:
Why are you using `setTimeout` here?

Reply:
Due to the way my search was working, but that will change, and so this will I guess.